applicationconfig/client/src/test/java/com/azure/applicationconfig/ConfigurationClientTest.java

```diff
@@ -159,105 +163,371 @@ private void cleanUpResources() {
         logger.info("Finished cleaning up values.");
     }
 
+    
```

Comment:
nit: extra new line

Reply:
There are a lot of trailing white spaces. You may need to run a "Trim trailing whitespace" on this.